frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java

```diff
@@ -34,8 +39,8 @@
     }
 
     @RestMapping(name = "json")
-    public Message getHelloMessage() {
-        return new Message("Hello, World!");
+    public Object getHelloMessage() {
+        return JSON_TEST ? msgjson : new Message("Hello, World!");

```

Comment:
Please revert this. This function should always instantiate a new object. Code for testing something different shouldn't be committed here.

Reply:
This is just a debugging code. The reason why the test data used to locate JSON is not high depends on whether it is IO problem or JSON parsing problem

Reply:
The next update will delete this code

Reply:
Unfortunately I’m not going to merge debugging code that breaks the test rules. It’s too easy to forget about that with the number of tests here. You’ll have to test these types of things locally.

Reply:
OK, I'll take it out now

Reply:
Thanks!